app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -382,6 +383,26 @@ def save_labels_inventory(entity, hashes, target = nil)
     store_ids_for_new_records(entity.labels, hashes, [:section, :name])
   end
 
+  def save_labels_and_tags_inventory(entity, hashes, target = nil)
+    save_labels_inventory(entity, hashes, target)
+    save_tags_generated_from_labels(entity, hashes, target)
+  end
+
+  # Must be called after object already got its labels.
+  # This is achieved by :labels_and_tags going through save_labels_and_tags_inventory().
+  # TODO: Drop the order requirement by using the :labels hash instead of entity.labels?
+  #   Would still need still save_labels_and_tags_inventory() to get that hash.
```

Comment:
Done: a2acda9e28f10e6537fa83ad58e4df2e4db0858b
I'm not entirely happy with it.
(1) duplicating the hash->labels logic of `save_labels_inventory`, perhaps imperfectly.  Especially `:resource_type => entity.class.base_class.name`.
(2) "abusing" the mapping API with duck typing.  I did add a test to kinda check it "ought to work" :-/

* TODO: I finally found the existing tests exercising `save_inventory_container.rb`, can now add some.

Kept the use of `:labels_and_tags` to trigger generating tags.
(The alternative I see is sending separate `:labels` and `:generate_tags_from_labels` with identical content all the way from refresh_parser.)